lib/resources/example_json/r4_examples_care_team.rb

```diff
@@ -0,0 +1,129 @@
+# frozen_string_literal: true
+
+module Cerner
+  module Resources
+
+    R4_CARE_TEAM_ENTRY ||= {
+      'resourceType': 'CareTeam',
+      'id': 'SA-1316020-2076906-7016273-0-0-0',
+      'meta': {
+        'versionId': '1',
+        'lastUpdated': '2016-03-10T13:43:45.000-06:00'
+      },
+      'text': {
+        'status': 'additional',
+        'div': '&lt;div xmlns="http://www.w3.org/1999/xhtml"&gt;&lt;p&gt;&lt;b&gt;Care Team&lt;/b&gt;&lt;/p&gt;&lt;p&gt;&lt;b&gt;Patient&lt;/b&gt;: new, testfa2&lt;/p&gt;&lt;p&gt;&lt;b&gt;Status&lt;/b&gt;: Active&lt;/p&gt;&lt;p&gt;&lt;b&gt;Category&lt;/b&gt;: Encounter-focused care team&lt&lt;/p&gt;&lt;p&gt;Encounter&lt;/b&gt;: 2076906&&lt;/p&gt;lt;p&gt;&lt;b&gt;Participant&lt;/b&gt;:&lt;/p&gt;&lt;ul&gt;&lt;li&gt;Name:, Parimi Vaishnavi; Role:Respiratory Therapy&lt;/li&gt;&lt;/ul&gt;&lt;/div&gt;' # rubocop:disable Layout/LineLength
+      },
+      'status': 'active',
+      "category": [
+        {
+          "coding": [
+            {
+              "system": 'http://loinc.org',
+              "code": 'LA27976-2',
+              "display": 'Encounter-focused care team'
+            }
+          ],
+          "text": 'Encounter-focused care team'
+        }
+      ],
+      "subject": {
+        "reference": 'Patient/1316020',
+        "display": 'new, testfa2'
+      },
+      "encounter": {
+        "reference": 'Encounter/2076906'
+      },
+      "participant": [
+        {
+          "role": [
+            {
+              "text": 'Respiratory Therapy'
+            }
+          ],
+          "member": {
+            "reference": 'Practitioner/7016273',
+            "display": ', Parimi Vaishnavi'
+          }
+        }
+      ]
+    }.freeze
+
+    R4_CARE_TEAM_BUNDLE ||= {
+      'resourceType': 'Bundle',
+      'id': '8c34294c-ba46-492d-bb8e-9937db0ddd5b',
+      'type': 'searchset',
+      'link': [
+        {
+          'relation': 'self',
+          'url': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/CareTeam?patient='\
```

Comment:
don't need to wrap text this way in my opinion

Reply:
Addressed. https://github.com/cerner/fhir.cerner.com/pull/470/commits/1c1aad456a43d4ba5ae6e7fc8620a05d19202987#diff-05cdc79578a87a8b1066e2bca824be5f

Reply:
apply to all line wraps

Reply:
Addressed https://github.com/cerner/fhir.cerner.com/pull/470/commits/31181ae509682c786080e6ef354f2c771f7a5bf2

Reply:
i don't see a line wrapping change in that commit

i removed the line wrapping with https://github.com/cerner/fhir.cerner.com/pull/470/commits/d035801ecfcbf522dfa51be6884f1011b582c086#diff-f7c10b53dd7ead0281cf4dbd1281a12cR15 and fixed the lint error with https://github.com/cerner/fhir.cerner.com/pull/470/commits/1822bc1d106921d7c6662d7648c00bc10b79d01c#diff-f7c10b53dd7ead0281cf4dbd1281a12cR15

i also added a change to a DSTU 2 file that used the wrong rubocop namespace since I noticed it getting flagged while checking lint